performances/performance_test/include/performance_test/ros2/lifecycle_node.hpp

```diff
@@ -244,6 +245,17 @@ template <typename> friend class System;
     return trackers;
   }
 
+  std::shared_ptr<Trackers> pub_trackers()
+  {
+    auto trackers = std::make_shared<Trackers>();
+
+    for(const auto& pub : _pubs)
+    {
+      trackers->push_back({pub.first, pub.second.second});
```

Comment:
What about adding some info about what is it being pushed to trackers? Like
```suggestion
      const auto & topic_name = pub.first;
      const auto & tracker = pub.second.second;
      trackers->push_back({topic_name, tracker});
```

Reply:
I would defer this to a more general refactor... A `Tracker` structure shouldn't be a pair made of a topic name and a tracker in the first place.